wren-ai-service/src/utils.py

```diff
@@ -1,3 +1,6 @@
+"""
+CAUTION: before running this code, please ensure the given dataset's mdl model is deployed already
+"""
```

Comment:
curiously, why did you add this comment, seems we didn't directly run any code in this module.